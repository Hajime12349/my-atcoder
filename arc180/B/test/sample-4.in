10 5
8 7 6 10 9 3 1 5 2 4
