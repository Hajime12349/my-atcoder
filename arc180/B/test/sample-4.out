15
3 8
2 8
3 10
3 9
1 8
2 10
2 9
2 7
1 10
5 10
1 9
4 10
4 9
1 7
1 6
